### PR TITLE
Menu Changes

### PR DESCRIPTION
**Pull request by kelnera**
I have to work through this, so you may see a lot of commits. I’m
trying to create new menu items, delete unnecessary menu items, and
rearrange things.

### CODE REVIEW

**Comment by ramereth** [comment]
@kelnera did you intend to remove the about page?

**Comment by kelnera** [comment]
I removed the about page thinking it would just remove the page with the
letter on it which we were planning on removing, but then realized too
little too late that took away the tab too. I'm working on it.

**Comment by kelnera** [comment]
@osu-cass/cass-media @alxngyn @leian7 @athai @subnomo Okay, I think this is right now. If I could have a lot of people look over this one though, that would be great.

File: content/pages/Partner-with-us/Partner-with-us.rst [hunk 1]
```diff
@@ -0,0 +1,28 @@
+Partner with Us
+===============
+:title: Partner with Us
+:slug: partner-with-us
+:menu: top, Partner with Us, 4
+
+From the Director:
+
+*Welcome to the Oregon State University Center for Applied Systems and
+Software-CASS.*
```
Comment (leian7) [inline content/pages/Partner-with-us/Partner-with-us.rst]: Should this be "... Software-- CASS", with the extra dash and space?

Comment (kelnera) [inline content/pages/Partner-with-us/Partner-with-us.rst]: I added the dash but not the space. But yeah it was probably supposed to be
like that. Good catch.

File: content/pages/Partner-with-us/Partner-with-us.rst [hunk 2]
```diff
@@ -0,0 +1,28 @@
+Partner with Us
+===============
+:title: Partner with Us
+:slug: partner-with-us
+:menu: top, Partner with Us, 4
```
Comment (athai) [inline content/pages/Partner-with-us/Partner-with-us.rst]: Shouldn't this be menu item 3?

Comment (kelnera) [inline content/pages/Partner-with-us/Partner-with-us.rst]: Yeah, so I put the correct weights for each tab (including Student
Experience) and for some reason they got mixed up. It's not really a big
deal so I wasn't going to try to figure out why it's doing that.

Comment (Kennric) [inline content/pages/Partner-with-us/Partner-with-us.rst]: I am not sure I understand - should this be 3 or 4? Is it good where it is?

Comment (kelnera) [inline content/pages/Partner-with-us/Partner-with-us.rst]: I realized it was a mistake on my part. I've fixed it now. It should all be
right.

**Comment by kelnera** [comment]
I want to push this one through first so if I could get some +1s here first, that would be great.

**Comment by Kennric** [comment]
This all looks fine to me. +1

@ramereth ?

**Pull request merged**